File: src/main/java/com/mybar/bartender/service/RoleService.java
package com.mybar.bartender.service;


import com.mybar.bartender.model.Role;
import com.mybar.bartender.repository.RoleRepository;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;

@Service
public class RoleService {
    @Autowired
    private RoleRepository roleRepository;

    public Role getUserRole() {
        return roleRepository.findByName("ROLE_USER").get();
    }
}

File: src/main/kotlin/com/mybar/bartender/service/RoleService.kt
package com.mybar.bartender.service

import com.mybar.bartender.model.Role
import com.mybar.bartender.repository.RoleRepository
import org.springframework.stereotype.Service

@Service
class RoleService(
    private var roleRepository: RoleRepository
) {
    fun getUserRole(): Role {
        return roleRepository.findByName("ROLE_USER")!!
    }
}